src/components/memorygame/MemoryGameCards/index.jsx

```diff
@@ -78,9 +78,7 @@ const MemoryGameCards = ({ setMatchedCards, matchedCards, setIsCompleted }) => {
       targetCardState.isFlipped = !targetCardState.isFlipped;
       setCardStates(() => [...copyOfCardStates]);
     } else {
-      throw new Error(
-        "Invalid property was passed to changeCardStatePropertyToOpposite"
-      );
+      throw new Error("invalid property");
```

Comment:
why was this made to be less descriptive?

Reply:
Oh I guess I reverted some changes by mistake. I will take a look

Reply:
PTAL